packages/react/src/themes/teams/components/Checkbox/checkboxStyles.ts

```diff
@@ -1,13 +1,104 @@
 import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
-import { CheckboxProps, CheckboxState } from '../../../../components/Checkbox/Checkbox'
+import Checkbox, { CheckboxProps, CheckboxState } from '../../../../components/Checkbox/Checkbox'
 import { CheckboxVariables } from './checkboxVariables'
 import getBorderFocusStyles from '../../getBorderFocusStyles'
 
 const checkboxStyles: ComponentSlotStylesInput<CheckboxProps & CheckboxState, CheckboxVariables> = {
   root: ({ props: p, variables: v, theme: t }): ICSSInJSStyle => ({
+    color: v.checkboxTextColor,
+    padding: v.rootPadding,
+
     ...getBorderFocusStyles({
       siteVariables: t.siteVariables,
       isFromKeyboard: p.isFromKeyboard,
+      borderRadius: '3px', // shouldn't default be correct?
```

Comment:
I had to modify the "universal" getBorderFocusStyles to make the radius of the corner match the redline. It seems like that should not be something that is necessary, is global getBorderFocusStyles correct?

Reply:
By default, in `siteVariables` the `borderRadius` is defined to be `2px`. If we need 3px only for the checkbox, that we may parametrize the `getBorderFocusStyles`, to receive this param, but the default can be the value of the siteVariables. Would that solve your issue?

Reply:
I looked into it more. getBorderFocusStyles produces the correct result in the case of button and dropdown, but the focus indicator in the redline for the checkbox does have more radius in the corners. I think just changing it as a one-off within checkboxStyles is fine for now.